ROMFS/px4fmu_common/init.d/rc.mavlink

```diff
@@ -162,6 +157,17 @@ if param compare SYS_COMPANION 3115200
 then
 	mavlink start -d ${MAVLINK_COMPANION_DEVICE} -b 115200 -r 1000 -f
 fi
+if param compare SYS_COMPANION 4115200
+then
+	usleep 200000 			# add a sleep here to make sure that the module is powered
+	if iridiumsbd start -d ${MAVLINK_COMPANION_DEVICE}
+	then
+		mavlink start -d /dev/iridium -m iridium -b 115200
+	else
+		echo "WARN: Iridiumsbd driver not started, reboot"
```

Comment:
Is the error message here really helpful? Who sees that in practice?

Reply:
I could remove it but still I think it might be helpful for debugging purpose.

Reply:
It would be much better if iridiumsbd itself has a PX4_ERR so it ends up in a log.

Reply:
They were warnings so far. So I changed them to errors and removed the echo in the startup script.